sudoku/src/domain/sudoku-model.ts

```diff
@@ -0,0 +1,98 @@
+const sudoku_example = [5, 3, 0, 0, 7, 0, 0, 0, 0, 6, 0, 0, 1, 9, 5, 0, 0, 0, 0, 9, 8, 0, 0, 0, 0, 6, 0, 8, 0, 0, 0, 6, 0, 0, 0, 3, 4, 0, 0, 8, 0, 3, 0, 0, 1, 7, 0, 0, 0, 2, 0, 0, 0, 6, 0, 6, 0, 0, 0, 0, 2, 8, 0, 0, 0, 0, 4, 1, 9, 0, 0, 5, 0, 0, 0, 0, 8, 0, 0, 7, 9];
+
+const SUDOKU_ROWS = 9;
+const SUDOKU_COLS = 9;
+const NUM_OF_SQUARES = SUDOKU_ROWS * SUDOKU_COLS;
+//SQUARES = list(range(NUM_OF_SQUARES))
+
+
+function printSudoku(sudoku: number[]): void {
+   for(let row=0; row<SUDOKU_ROWS ;row++){
+      let s = '';
+      let delimiter = '------+-------+------';
+      for(let col=0; col<SUDOKU_COLS ;col++){
+         if (sudoku_example[row*SUDOKU_ROWS+col] == 0) {
+            s +='  '
+         } else {
+            s += sudoku_example[row*SUDOKU_ROWS+col].toString()+' ';
+         }                  
+         if((col+1)%3 === 0 && (col+1)<SUDOKU_COLS){
+            s+='| ';
+         }
+      }
+      console.log(s);
+      if((row+1)%3 === 0 && (row+1)<SUDOKU_ROWS){
+            console.log(delimiter);
+         }  
+   }
+}
+
+function get_row_number(index: number): number {
```

Comment:
Tror det vanligaste är att skriva funktionernas namn i camel case, så har i alla fall varig fallet i alla JavaScript projekt som jag jobbat med. 😉

```suggestion
function getRowNumber(index: number): number {
```
Exempelvis så varnar min IDE om jag försöker definiera en konstant `const talel_dayekh`. Detta kan vi konfigurera med ESLint och Prettier.

<img width="530" alt="Screenshot 2021-04-24 at 21 06 51" src="https://user-images.githubusercontent.com/43838780/115970282-ad119500-a541-11eb-866d-9eb905b54da8.png">